scripts/summit_bsub_openmc_nekrs.sh

```diff
@@ -0,0 +1,116 @@
+#!/usr/bin/env bash
+set -e
+
+if [ $# -ne 3 ]; then
+  echo "usage: $0 <casename> <number of compute nodes> <hh:mm>"
+  exit 1
+fi
+
+if [ ! -d "$NEKRS_HOME" ]; then
+  echo "ERROR: The env variable NEKRS_HOME does not point to an existing directory"
+  exit 1
+fi
+
+if [ -z "$PROJ_ID" ]; then
+  echo "ERROR: The env variable PROJ_ID is not set"
+  exit 1
+fi
+
+: ${OCCA_CACHE_DIR:="$PWD/.cache/occa"}
+NVME_HOME="/mnt/bb/$USER/"
+XL_HOME="/sw/summit/xl/16.1.1-3/xlC/16.1.1"
+
+export NEKRS_HOME
+export OCCA_CACHE_DIR
+export NEKRS_HYPRE_NUM_THREADS=1
+export OGS_MPI_SUPPORT=1
+export OCCA_CXX="$XL_HOME/bin/xlc" 
+export OCCA_CXXFLAGS="-O3 -qarch=pwr9 -qhot -DUSE_OCCA_MEM_BYTE_ALIGN=64" 
+export OCCA_LDFLAGS="$XL_HOME/lib/libibmc++.a"
+
+#export OCCA_VERBOSE=1
+#export OMPI_LD_PRELOAD_POSTPEND=$OLCF_SPECTRUM_MPI_ROOT/lib/libmpitrace.so
+
+#export PAMI_ENABLE_STRIPING=1
+#export PAMI_IBV_ADAPTER_AFFINITY=1
+#export PAMI_IBV_DEVICE_NAME="mlx5_0:1,mlx5_3:1"
+#export PAMI_IBV_DEVICE_NAME_1="mlx5_3:1,mlx5_0:1"
```

Comment:
Are these being left commented because they're expected to be needed at some point in time? Probably either leave a comment here or remove them.

Reply:
I'll remove them.  They're in the script from the nekRS repo, but as far as I know, we don't use them.

Reply:
Fixed in 6cd0a6ff41a0375bcdb2257f15b9ece4508e97cd